css/lean.css

```diff
@@ -1,4 +1,13 @@
 @charset "UTF-8";
+
+li.nav-item.dropdown:hover .dropdown-menu {
+  display: block;
+}
+
+.dropdown-menu {
+  margin-top: -1px !important; /* When the dropdown comes down on hover, this will make it easier to get to the dropdown contents without losing it. */
```

Comment:
Is the `!important` necessary?

Reply:
yep

Reply:
Sorry, by "necessary", I don't mean "does it fail to work if you remove `!important` here?". I mean: are you sure there is no way to get the desired effect without using it?

(I'm far from a CSS expert, so I'm just recalling recommendations like [this one](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) to avoid `!important` in most cases if possible.)

Reply:
Bootstrap sets a margin-top by default, so I have to override it.

There might be a way to avoid it, but CSS order of rules is a big mess and I don't like to mess with it.

Reply:
Yes, I just had a look and you're right. Thanks.